config_management/domains/views.py

```diff
@@ -0,0 +1,70 @@
+from django.shortcuts import render, redirect, HttpResponse
+from django.views.generic import View, TemplateView
+from .models import DomainName
+# from .forms import AddDomainForm
+from django.http.response import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+import json
+
+# Create your views here.
+
+def list_all_domains(request):
+    if request.method == 'GET':
+
+        temp = DomainName.objects.all()
+        domain_list = []
+        for i in temp:
+            domain_list.append((i.name, i.id))
+
+        args = {'listing all domains':'ok', 'status_code':200, 'list of all domains':domain_list}
```

Comment:
Follow variable name convection, no spaces please.
[Comment for all other places as well]